docker/test/docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-version: "3.4"
+version: "3.7"
```

Comment:
I think this is no longer needed if we use the Compose Specification:
https://docs.docker.com/compose/compose-file/compose-versioning/#versioning
https://docs.docker.com/compose/compose-file/04-version-and-name/

Considering we're using `ubuntu-latest` Github Action runner, which is Ubuntu 22.04, we already have docker-compose 1.29 which supports the docker compose specification:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#tools

We should probably be able to remove the version and upgrade the requirement in README to 19.03:
https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix
https://github.com/eclipse/hara-ddiclient/blob/master/README.md?plain=1#L53

Reply:
Made the changes in a new commit.